ldes-server-retention/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/retention/services/retentionpolicy/execution/RetentionServiceTest.java

```diff
@@ -55,9 +57,9 @@ void when_MembersOfFragmentMatchRetentionPoliciesOfView_MembersAreRemovedFromThe
 
 		retentionService.executeRetentionPolicies();
 
-		verify(memberPropertiesRepository).removeExpiredMembers(VIEW_A, timeBasedRetentionPolicy);
-		verify(memberPropertiesRepository).removeExpiredMembers(VIEW_B, versionBasedRetentionPolicy);
-		verify(memberPropertiesRepository).removeExpiredMembers(VIEW_C, timeAndVersionBasedRetentionPolicy);
+		verify(memberPropertiesRepository).findExpiredMembers(VIEW_A, timeBasedRetentionPolicy);
+		verify(memberPropertiesRepository).findExpiredMembers(VIEW_B, versionBasedRetentionPolicy);
+		verify(memberPropertiesRepository).findExpiredMembers(VIEW_C, timeAndVersionBasedRetentionPolicy);
```

Comment:
Shouldn't pageMemberRepo.remove or something like that not be verified here as well?